ToyBox/classes/Infrastructure/LootHelper.cs

```diff
@@ -117,33 +123,45 @@ public static class LootHelper {
                 revealer.RunAction();
             }
         }
+        public static void updateHidden() {
+            // root -> InGamePCView(Clone) -> InGameStaticPartPCView
+            var MarksLoot = SceneManager.GetSceneByName("UI_Ingame_Scene").GetRootGameObjects()[1].findChild("InGameStaticPartPCView")
```

Comment:
Is there any reason you're not using the `/` syntax here instead? e.g. "InGameStaticPartPCView/StaticCanvas/ServiceWindowsPCView/Background/Windows/LocalMapPCView/ContentGroup/MapBlock/Map/MarksLoot"?